kernel.rb

```diff
@@ -171,4 +171,16 @@ def yield_self
   def Float(arg, exception: true)
     Primitive.rb_f_float(arg, exception)
   end
+
+  #
+  # call-seq:
+  #    Kernel.fork  [{ block }]   -> integer or nil
+  #
+  # Creates a subprocess.
+  #
+  # See Process.fork
+  #
+  def fork(*args, **kwargs, &block)
+    Process.fork(*args, **kwargs, &block)
+  end
```

Comment:
I wanted to use `...` argument forwarding here, but apparently that's not possible?

```
making kernel.rbinc
Traceback (most recent call last):
	4: from ./tool/mk_builtin_loader.rb:364:in `<main>'
	3: from ./tool/mk_builtin_loader.rb:364:in `each'
	2: from ./tool/mk_builtin_loader.rb:366:in `block in <main>'
	1: from ./tool/mk_builtin_loader.rb:247:in `mk_builtin_header'
./tool/mk_builtin_loader.rb:247:in `compile': <compiled>:183: syntax error, unexpected ..., expecting ')' (SyntaxError)
  def fork(...)
           ^~~
<compiled>:184: syntax error, unexpected ..., expecting ')'
    Process.fork(...)
                 ^~~
make: *** [kernel.rbinc] Error 1
```

Reply:
Actually I had to move it back to `process.c` to be able to alias it as `rb_f_notimplement` on Windows.